mitodl/mitxpro
# Do not log an error if HUBSPOT_CREATE_USER_FORM_ID is unset; sync hubspot contact on login

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2660 

#### What's this PR do?
- Removes the error log statement if HUBSPOT_CREATE_USER_FORM_ID is not set
- Syncs contact w/hubspot on login to reflect any recent changes

#### How should this be manually tested?
- Set `HUBSPOT_CREATE_USER_FORM_ID` to the same value as RC in your .env file
- Follow the instructions for https://github.com/mitodl/mitxpro/pull/2653
- Log out and back in a couple times.
- There should not be any errors in the log.
- Repeat the above but with `HUBSPOT_CREATE_USER_FORM_ID` removed from your .env file

Related:
